recipes/hips/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hips" %}
+{% set version = "0.2" %}
+{% set sha256 = "e9c533e2c3f6feb63c8fa83f23c1ad2f31036afd873bbd352a705c866a3dbdcb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py<36]"
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy>=1.11
```

Comment:
I think you need a space between the name and version here and below

Reply:
Done.